### PR TITLE
Adding react redux example

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

**Comment by bertrandk** [comment]
Awesome. LGTM :ship:

File: examples/react-redux/.babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015", "react", "stage-0"]
```
Comment (jhabdas) [inline examples/react-redux/.babelrc]: Is this actually using anything from Stage 0? If not, I'd using Stage-1 as the Strawman proposal is a bit weak IMO.

**Comment by jhabdas** [comment]
Good stuff! An example would've been nice when I did my integration using this module, but there's a lot of boilerplate here and examples directories are hard to maintain because they're like maintaining an app within a library. Would it make more sense to move this to a separate seed repo similar to what I did with [my RN starter kit](https://github.com/jhabdas/react-native-webpack-starter-kit)?

File: examples/react-redux/.babelrc (hunk 1 above)
Comment (sumitarora) [inline examples/react-redux/.babelrc]: @jhabdas it's not currently and it's just sample to tell how to configure you can use any setup you want

**Comment by sumitarora** [comment]
@jhabdas There's not much in examples currently, good point though we might go with that option once we have a more elaborate example with all event types etc.

**Comment by jhabdas** [comment]
:+1:

**Pull request merged**